src/components/Dashboard/BarChart/BarChart.jsx

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import { HorizontalBar, defaults} from 'react-chartjs-2';
```

Comment:
noticed there's no space padding after ```defaults``` like there is before ```HorizontalBar``` 

Instead of doing this by hand install ```prettier``` into your editor and it'll fix up all the formatting for you